### PR TITLE
Adding `soleimani.hackclub.com`

### PR DESCRIPTION
**Pull request by soleimanis**
# Adding `soleimani.hackclub.com`

## Description

Hi, I hope you're having a wonderful day!

This is a showcase of Hack Club projects and accomplishments.

Thank you very much in advance.

### CODE REVIEW

**Comment by Muirrum** [comment]
What's the difference between this and #1292 ?

**Comment by soleimanis** [comment]
There is absolutely no difference. It seems I previously thought I didn't want to move forward with it and deleted and closed the pull request but I had the same idea again and forgot I had already created it.
Apologies for creating a duplicate pull request.

**Comment by Muirrum** [comment]
As I said in the previous PR, personal sites are out-of-scope for hackclub.com. Please move this to dino.icu

File: hackclub.com.yaml [hunk 1]
```diff
@@ -2483,6 +2483,10 @@ smus:
   - ttl: 600
     type: CNAME
     value: smushackclub.github.io.
+soleimani:
+  - ttl: 600
+    type: CNAME
+    value: soleimanis.github.io.
```
Comment (Muirrum) [inline hackclub.com.yaml:2489]: out-of-scope, please move to dino.icu

**Review by Muirrum — CHANGES_REQUESTED** [review]

File: hackclub.com.yaml (hunk 1 above)
Comment (soleimanis) [reply hackclub.com.yaml:2489]: Will do, thank you very much

**Pull request closed without merging**